internal/kibana/space_test.go

```diff
@@ -39,6 +39,15 @@ func TestAccResourceSpace(t *testing.T) {
 					resource.TestCheckResourceAttrSet("elasticstack_kibana_space.test_space", "image_url"),
 				),
 			},
+			{
```

Comment:
This test will only pass if we're running against a supported Elastic Stack version (8.18 or higher). Use `versionutils.CheckIfVersionIsUnsupported` as a `SkipFunc` to only run this test when the `solution` field is supported.